<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>寰宇医桥</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    /* 主窗口渐变背景 */
    QWidget#Widget {
        background-color: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #d0f0ff,
            stop:1 #ffffff
        );
    }

    /* 标题样式 */
    #titleLabel {
        color: #1e88e5;
        font-size: 36px;
        font-weight: bold;
        text-shadow: 1px 1px 3px #81d4fa;
    }

    /* 欢迎标签 */
    #welcomeLabel {
        color: #1e88e5;
        font-size: 20px;
        font-weight: bold;
        qproperty-alignment: AlignCenter;
    }

    /* 输入框样式 */
    QLineEdit {
        border: 2px solid #81d4fa;
        border-radius: 12px;
        padding: 10px;
        font-size: 14px;
        background-color: #ffffff;
        color: #0f1f38;
        selection-background-color: #4fc3f7;
        selection-color: #ffffff;
    }
    QLineEdit:focus {
        border: 2px solid #1e88e5;
    }

    /* 按钮高级渐变科技风 */
    QPushButton {
        border-radius: 14px;
        border: 1px solid #64b5f6;
        color: #ffffff;
        font-weight: bold;
        font-size: 16px;
        padding: 10px 20px;
        background-color: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #6ec6ff,
            stop:1 #1e88e5
        );
    }
    QPushButton:hover {
        background-color: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #81d4fa,
            stop:1 #1976d2
        );
    }
    QPushButton:pressed {
        background-color: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #1e88e5,
            stop:1 #1565c0
        );
    }
    QPushButton {
        text-shadow: 1px 1px 2px rgba(0,0,0,0.2);
    }

    /* 急诊按钮 */
    #emergencyButton {
        border-radius: 14px;
        border: 1px solid #e57373;
        color: #ffffff;
        background-color: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #ff7f7f,
            stop:1 #e53935
        );
    }
    #emergencyButton:hover {
        background-color: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #ff8a80,
            stop:1 #f44336
        );
    }
    #emergencyButton:pressed {
        background-color: qlineargradient(
            x1:0, y1:0, x2:0, y2:1,
            stop:0 #e53935,
            stop:1 #c62828
        );
    }

    /* 单选按钮样式 */
    QRadioButton {
        font-size: 14px;
        spacing: 8px;
        color: #1e88e5;
    }

    /* 表单容器样式 */
    #formContainer {
        background-color: transparent;
        border-radius: 18px;
        padding: 20px;
        margin: 20px;
        border: 1px solid transparent;
    }
   </string>
  </property>
  <widget class="QWidget" name="formContainer" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>400</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="welcomeLabel">
      <property name="text">
       <string>欢迎登录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="usernameLineEdit">
      <property name="placeholderText">
       <string>请输入用户名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>请输入密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="roleLabel">
      <property name="text">
       <string>请选择用户类型：</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="doctorRadioButton">
        <property name="text">
         <string>医生</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="patientRadioButton">
        <property name="text">
         <string>患者</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="loginButton">
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="signupButton">
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="emergencyButton">
      <property name="text">
       <string>急诊通道</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_navigation">
      <property name="text">
       <string>医院导航</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitButton">
      <property name="text">
       <string>退出系统</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>243</width>
     <height>92</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="logoLabel">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="pixmap">
       <pixmap>logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>寰宇医桥</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
